README.md

```diff
@@ -1,209 +1,177 @@
-# Sendbird UIKIT React
+# [Sendbird](https://sendbird.com) UIKit for React
 
-Find the docs at: https://docs.sendbird.com/javascript/ui_kit_getting_started
+![Platform](https://img.shields.io/badge/platform-JAVASCRIPT-orange.svg)
+![Languages](https://img.shields.io/badge/language-JAVASCRIPT-orange.svg)
+[![npm](https://img.shields.io/npm/v/sendbird.svg?style=popout&colorB=red)](https://www.npmjs.com/package/sendbird)
 
-For bugs and feature requests, visit: https://community.sendbird.com/
+## Table of contents
 
-Samples: https://github.com/sendbird/SendBird-JavaScript/tree/master/uikit-samples
+  1. [Introduction](#introduction)
+  1. [Before getting started](#before-getting-started)
+  1. [Getting started](#getting-started)
+  1. [UIKit at a glance](#uikit-at-a-glance)  
 
-## Install
+<br />
 
-`npm i sendbird-uikit`
+## Introduction
 
-> Note - If you are building for IE11, import an Array.prototype.find polyfill - https://www.npmjs.com/package/array.prototype.find or https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#Polyfill
+**Sendbird UIKit** for React is a development kit that enables fast and easy integration of standard chat features into new or existing applications. From the overall theme to individual styles such as colors and fonts, components can be fully customized to create an in-app chat experience unique to your brand identity.
 
-## Available Components
+> **Note**: Currently, UIKit for iOS supports group channels only.
 
-The components that you receive once you install the uikit
+### Benefits
 
-* SendBirdProvider: The context provider that stores SDK and user information
-* withSendBird: The Higher Order Component to access data from SendBirdProvider
-* ChannelList: UI component to render a list of components
-* Channel: A UI Component where conversations happen
-* ChannelSettings: A component to change various settings of a given channel
-* App: A full fledged app component made by combining the above components
+- Easy installation
+- Fully-featured chat with a minimal amount of code
+- Customizable components, events, and views
+- Customizable user list to enable chat among specified users
 
-## Usage
+![ThemeLight](https://static.sendbird.com/docs/uikit/javascript/theme-light_20200401.png)
 
-There are two usages
+### More about Sendbird UIKit for React
 
-### Simple
+Find out more about Sendbird UIKit for React on [UIKit for React doc](https://sendbird.com/docs/uikit/v1/javascript/getting-started/about-uikit). If you have any comments or questions regarding bugs and feature requests, visit [Sendbird community](https://community.sendbird.com). 
 
+<br />
+
+## Before getting started
+
+This section shows the prerequisites you need to check to use Sendbird UIKit for React.
+
+### Requirements 
+
+The minimum requirements for Sendbird UIKit for React are:
+
+- `React 16.8.0+`
+- `React DOM 16.8.0+`
+- `Sendbird Chat SDK for JavaScript 3.0.115+`
+- `moment 2.24.0+`
```

Comment:
This part changed in last update see - https://github.com/sendbird/SendBird-Docs/pull/1807/files#diff-0b117f748099111cea457d476d04263e2dd0740f7b457f097246a9b8b1e29075L27-R30

Reply:
done!